### DISCUSSION TITLE
How do I get folder names in the nav to be capitalized?

### DISCUSSION THREAD

**Discussion by redonkulus** [shuding/nextra#2475] [Q&A]
I've checked for this topic in past Issues and Discussions but have not seen the question. I have a directory structure like this:

```text
/pages
  -> /foo
     -> bar.mdx
```

The results nav is lowercase `foo -> Bar`. However, I would like the `foo` directory name to be capitalized (e.g. `Foo`).

I know I can use CSS to style it or capitalize the folder name. I do not want to capitalize it because that also changes the URL case (which I want to keep lowercase). I'm migrating from an existing project which uses all lowercase already.

Is there a setting to enable this?

**Comment by dyoungwd2** [comment]
create _meta.json file where u can set the names howyou want them to appear The way i have done it is in/pages a _meta.json then in the sub folders add another for the files names within that folder.

For example in the root


```
{
"folder1 ": FOLDER  1",

"folder2 ":  "FOLDER  2"


  }

```

and in the folders


 ```
{
 "page1 ": "PAGE 1",
  "page2":  "PAGE2"
 
}

```

**Reply by redonkulus** [reply]
Thanks for the reply. The `_meta.json` is powerful, but it also requires me to rewrite the directory/file structure just to rename one directory. For example, if I want to rename one directory that is at the bottom of the directory list, I have to list them all otherwise the one I renamed gets moved to the top. 

A simple config like `navigation.upperCase: true` would simplify a lot.
